examples/angular/src/app/pages/components/loading/loading.component.ts

```diff
@@ -0,0 +1,167 @@
+import { Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-loading',
+  standalone: true,
+  imports: [CommonModule],
+  template: `
+    <div class="loading-container">
+      <!-- Layout 1 -->
+      <ng-container *ngIf="selectedLayout === 1">
```

Comment:
Why do we use new syntax and old syntax in some parts? I suggest to use one style in all examples.

Reply:
Because this was written by AI 🤣